pkg/apis/options/providers.go

```diff
@@ -15,6 +17,8 @@ var OIDCAudienceClaims = []string{"aud"}
 type Providers []Provider
 
 // Provider holds all configuration for a single provider
+// NOTICE: bcs unset booleans default to false when unmarshalled from yaml,
+// NOTICE: ptrs to bool are used where the default is non-false -> true
```

Comment:
As these comments end up in the generated docs, I don't think we need these. I think this is a common enough pattern in Go that users will understad